st2common/bin/paramiko_ssh_evenlets_tester.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python2.7
```

Comment:
It's definitely a step above that - https://github.com/StackStorm/st2/commit/d97cdf67ca62459a32a5ab08806d4054fe6b58bc :P